### PR TITLE
Clean up docs

### LINKED ISSUE #15: move narrative and example from API to main docs
The whole docs section on what waterdynamics does and how to use it is not API documentation but proper documentation. It should be moved to its own page and appear before the API docs.

### PR DESCRIPTION
**Pull request by ianmkenney**
Fixes #15

PR Checklist
------------
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

**Review by IAlibay — APPROVED** [review]
This looks great, thanks!

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -1,23 +1,46 @@
...
+or with :program:`mamba` (if you have it installed):
+
+.. code-block:: sh
+
+	mamba install waterdynamics
+
+From the Python Package Index
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The waterdyanmics package can be installed from PyPI with:
```
Comment (orbeckst) [inline docs/source/getting_started.rst]: waterdynamics

File: docs/source/index.rst
Comment (orbeckst) [inline docs/source/index.rst:1]: Add the intro from https://docs.mdanalysis.org/stable/documentation_pages/analysis/waterdynamics.html#water-dynamics-analysis-mdanalysis-analysis-waterdynamics , it contains the summary _and_ the citations (which are currently missing).

File: docs/source/examples.rst [hunk 2]
```diff
@@ -0,0 +1,267 @@
+Class Example Usage
+===================
+
+``waterdynamics`` operates around class based analyses.
+Find examples of their use below.
+
```
Comment (orbeckst) [inline docs/source/examples.rst:17]: include the SeeAlso about h-bonds from https://docs.mdanalysis.org/stable/documentation_pages/analysis/waterdynamics.html#hydrogenbondlifetimes

File: docs/source/examples.rst [hunk 3]
```diff
@@ -0,0 +1,267 @@
+Class Example Usage
```
Comment (orbeckst) [inline docs/source/examples.rst]: I'd just call it "Examples"

File: docs/source/index.rst [hunk 4]
```diff
@@ -13,6 +13,7 @@ This package provides classes and functions for the analysis of water dynamics a
    :caption: Contents:
 
    getting_started
+   examples
```
Comment (orbeckst) [inline docs/source/index.rst:20]: add a section after examples about *output*

**Review by orbeckst — CHANGES_REQUESTED** [review]
- include introduction *with citations*
- include section on *output* https://docs.mdanalysis.org/stable/documentation_pages/analysis/waterdynamics.html#output (as in the original docs)
- see small comments inline

File: docs/source/index.rst (hunk 4 above)
Comment (ianmkenney) [reply docs/source/index.rst:20]: The original contents of the output section were put in to the examples subsections.

Comment (orbeckst) [reply docs/source/index.rst:20]: Make it a separate section — it's important enough.

Comment (ianmkenney) [reply docs/source/index.rst:20]: Moved

**Review by orbeckst — APPROVED** [review]

**Comment by orbeckst** [comment]
Thanks @ianmkenney !

**Pull request merged**